app/assets/javascripts/components/signup/signup.js.coffee

```diff
@@ -80,7 +83,11 @@ modulejs.define 'components/signup/signup',
         ),
         (div {className: 'side-info'},
           if studentData
-            (StudentRegistrationCompleteSideInfo {})
+            # StudentRegistrationCompleteSideInfo contains a login form
+            # If the student is already logged in because of the SSO path, don't show
+            # this form
+            if anonymous
+              (StudentRegistrationCompleteSideInfo {})
```

Comment:
Yeah that was might intention. I'll update the comment to make that more clear.  Probably we should put something there, but I wasn't sure what. This path is not currently used by anyone as far as I know.  But we are planning to start using it with Google SSO soon, so hopefully we can figure out what to put there.